package.json

```diff
@@ -34,10 +34,10 @@
 		"support-basic.json"
 	],
 	"devDependencies": {
-		"eslint-config-wikimedia": "0.21.0",
-		"postcss-less": "5.0.0"
+		"eslint-config-wikimedia": "0.21.0"
 	},
 	"dependencies": {
+		"postcss-less": "5.0.0",
```

Comment:
What's wrong with 6.0.0? It seemed to work fine in https://gerrit.wikimedia.org/r/c/mediawiki/extensions/VisualEditor/+/762472